src/components/EventCard.tsx

```diff
@@ -28,97 +21,52 @@ const EventCard = ({
   imageUrl,
   endDate,
   startDate,
-}: EventCardProps) => {
+}) => {
   const { locale } = useRouter()
   const { t } = useTranslation("page-community")
+
   const formatedDate = new Intl.DateTimeFormat(locale, {
     day: "2-digit",
     month: "short",
   }).formatRange(
-    // .replace(/-/g, "/") ==> Fixes Safari Invalid date
     new Date(startDate?.replace(/-/g, "/")),
     new Date(endDate?.replace(/-/g, "/"))
   )
 
   return (
-    <Box
-      className={className}
-      position="relative"
-      mt="0"
-      _before={clearStyles}
-      _after={clearStyles}
-      w="full"
-      h="full"
+    <Card
+      className={`border-lightBorder flex h-full flex-col rounded-md border ${className}`}
```

Comment:
Ah, what did you mean with `lightBorder`?

Reply:
No idea, just going to remove lol